foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java

```diff
@@ -116,9 +116,6 @@ public static ConcurrentCompositeConfiguration createLocalConfig() {
     }
 
     LOGGER.info("create local config:");
-    for (ConfigModel configModel : loader.getConfigModels()) {
-      LOGGER.info(" {}.", configModel.getUrl());
-    }
 
```

Comment:
Maybe it's not a good idea to remove this log.
This log is often checked when config related problems occur.

Reply:
Users decide whether to print through configuration